de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/75-de.metas.fresh.reports/5518792_sys_gh5125_AddFunction_Docs_Manufacturing_Order_Details.sql

```diff
@@ -0,0 +1,45 @@
+DROP FUNCTION IF EXISTS de_metas_endcustomer_fresh_reports.Docs_Manufacturing_Order_Details( IN numeric, IN numeric, IN Character Varying );
+
+CREATE OR REPLACE FUNCTION de_metas_endcustomer_fresh_reports.Docs_Manufacturing_Order_Details
+  (IN record_id   numeric,
+   IN p_m_attribute_id numeric,
+   IN p_ad_language character Varying)
+
+  RETURNS TABLE
+  (
+    line            numeric,
+    qtyrequiered    numeric,
+    uomsymbol       character varying,
+    value           character varying,
+    vendorProductNo character varying,
+    description     character varying,
+	attributes     character varying
+  )
+AS
+$$
+
+SELECT
+  line,
+  qtyrequiered,
+  COALESCE(uom.UOMSymbol, uomt.UOMSymbol) AS UOMSymbol,
+  p.value,
+  coalesce(bpp.productno, p.value)        as vendorProductNo,
+  coalesce(pt.description, p.description) as description,
+  Attributes.attributes_value
+FROM PP_Order_BOMLine bomLine
+
+  -- Product and its translation
+  JOIN M_Product p ON bomLine.m_product_id = p.m_product_id AND p.isActive = 'Y'
```

Comment:
if the product was deactivated since the PP_Order was created, do we really want the bomline to be not displayed in the the jasper